python/lsst/pipe/tasks/drpAssociationPipe.py

```diff
@@ -249,9 +249,9 @@ def visitDetectorPair(dataRef):
                 ssCatRef = ssObjectIdDict[key]
                 ssCat = ssCatRef.get()
                 finalVisitSummary = finalVisitSummaryIdDict[key[0]].get()
-                visitInfo = finalVisitSummary[0].visitInfo
-                bbox = finalVisitSummary[0].getBBox()
-                wcs = finalVisitSummary[0].wcs
+                visitInfo = finalVisitSummary.find(key[1]).visitInfo
```

Comment:
As we discussed I found this section hard to follow--a couple of comments describing the control flow would be helpful for future maintainers.